.pkg/FLATPAK/com.github.jpochyla.Psst.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.jpochyla.Psst</id>
+  <url type="homepage">https://github.com/jpochyla/psst</url>
+
+  <name>Psst</name>
+  <summary>Fast Spotify client with native GUI, without Electron, built in Rust.</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <description>
+    <p>
+      Fast Spotify client with native GUI, without Electron, built in Rust. Very early in development, lacking in features, stability, and general user experience. It is being tested only on Mac so far, but aims for full Windows and Linux support. Contributions welcome!
```

Comment:
Can we change this to the `bundle` values from Cargo.toml?

Reply:
Sadly not. Flatpak uses [AppStream](https://wiki.debian.org/AppStream).

Reply:
Oh sorry, I mean literally copy-pasting the texts from Cargo.toml.

Reply:
I'm quite unfamiliar with Rust in general (but familiar with Flatpak packaging). When using Cargo.toml, does it create a metadata file automatically?